### DISCUSSION TITLE
Support antd v5 ConfigProvider

### DISCUSSION THREAD

**Discussion by domatosource** [xrutayisire/react-js-cron#44] [Ideas]
Hi is it possible to add support for antd v5 themes preset algorithms?
https://ant.design/docs/react/customize-theme#use-preset-algorithms

**Comment by xrutayisire** [comment]
Hello,

Can you tell me a bit more about what you would need? What is your use case after the implementation?

**Comment by domatosource** [comment]
Basically I am using the code from the link in the previous post
```
const App: React.FC = () => (
  <ConfigProvider
    theme={{
      algorithm: theme.darkAlgorithm,
    }}
  >
    <Cron />
  </ConfigProvider>
);
```

and when I do a <Cron ....> the css gets all wonky, the select boxes renders either with white background, or black background with black text. I believe that is because with the ConfigProvider, the colours for the css classes ant-selector gets changed on the fly via code.

It would be good if perhaps <Cron /> takes in an additional prop that lets us select what preset theme we are using, and render the select boxes based on the theme.

**Reply by xrutayisire** [reply]
I see, good example, it's even a bug for me since theme management in antd is reason why they switch to CSS in JS for v5.

Do you think you could help on that? I don't know when I will have the time to do it. For sure not before next month at best :/

**Reply by domatosource** [reply]
Sure I can take a shot at it, but like you, not these few days. Perhaps next week or so.

**Comment by nefedov-dm** [comment]
Good day! I created PR: https://github.com/xrutayisire/react-js-cron/pull/52
It is about this discussion, can you check it please?
